PowerPointLabs/PowerPointLabs/HighlightBulletsText.cs

```diff
@@ -202,8 +205,11 @@ private static List<int> GetUnselectedParagraphs(Office.TextRange2 textRange, Of
                 {
                     continue;
                 }
+
+                bool hasCarriageReturn = textRange.Paragraphs[i].Text.EndsWith("\r");
+
                 if (selectedText.Start + selectedText.Length < paragraph.Start ||
-                    selectedText.Start > paragraph.Start + paragraph.Length - 1)
+                    selectedText.Start > paragraph.Start + paragraph.Length - 1 + (hasCarriageReturn ? 0 : 1))

```

Comment:
Yeah I was thinking of that too but I couldn't come up with a name (`lengthToAdd`? `carriageReturnOffset`? the name needs to capture the `if (ends with \r) value = 1` relationship). Any ideas?